src/styles/saas-list.less

```diff
@@ -14,21 +14,37 @@
   .card-view-pf .card-content {
     background-color: @color-pf-white;
     cursor: pointer;
+    margin: 0;
+
+    @media (min-width: @screen-sm-min) {
+      margin: 2px 0 10px 0;
+    }
   }
 
   .card-view-pf .card {
-    height: 200px;
-    width: 260px;
+    height: 100%;
+    width: 100%;
```

Comment:
I removed them and the cards remained their original 'large' height and width.  I then set them to 'inherit' and the height was ok, but the width fit the width of the content and didn't stretch to the end.